### PR TITLE
Wb 64 measuring impact

### PR DESCRIPTION
**Pull request by singhpritam18**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix # https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-64

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://wb-64-measuring-impact--world-bank--aemsites.hlx.page/en/drafts/pritam/home-page
https://wb-64-measuring-impact--world-bank--aemsites.hlx.page/en/drafts/pritam/who-we-are

### CODE REVIEW

File: blocks/measuring-impact-cards/measuring-impact-cards.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  if (indicatorTag && indicatorTag.indexOf('-') < 0) return indicatorTag;
+
+  const indicatorKey = indicatorTag.split('-')[0].trim().replace(/[^a-zA-Z0-9]/g, '');
+  return placeholdersData[indicatorKey] ? placeholdersData[indicatorKey] : indicatorKey;
+};
+
+const getColorCode = (indicatorTag, placeholdersData) => {
+  if (indicatorTag && indicatorTag.indexOf('-') < 0) return '#07AB50;';
+  const indicatorColorKey = indicatorTag.split('-')[1].trim().replace(/[^a-zA-Z0-9]/g, '');
+  return placeholdersData[indicatorColorKey] ? placeholdersData[indicatorColorKey]
```
Comment (mkbansal1) [inline blocks/measuring-impact-cards/measuring-impact-cards.js:15]: @singhpritam18  use prefix in keys to add context of block

File: blocks/measuring-impact-cards/measuring-impact-cards.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { div, a, p } from '../../scripts/dom-helpers.js';
+import { moveInstrumentation, fetchLanguagePlaceholders } from '../../scripts/scripts.js';
+
+const getShortendIndicatorName = (indicatorTag, placeholdersData) => {
+  // Return Same indicator tag content if hyphen is not present
+  if (indicatorTag && indicatorTag.indexOf('-') < 0) return indicatorTag;
```
Comment (mkbansal1) [inline blocks/measuring-impact-cards/measuring-impact-cards.js:6]: move generic code in a function to return array
` if (indicatorTag && indicatorTag.indexOf('-') < 0) return indicatorTag;

  const indicatorKey = indicatorTag.split('-')[0].trim().replace(/[^a-zA-Z0-9]/g, '');`

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
@singhpritam18 Added some minor comments, please incorporate

File: blocks/measuring-impact-cards/measuring-impact-cards.js (hunk 1 above)
Comment (singhpritam18) [reply blocks/measuring-impact-cards/measuring-impact-cards.js:15]: What's the expectation here ?
 
do you mean to have color keys like verticalname-color e.g. planet-green.
If yes, idea behind to have vertical color code authorable, is to give flexibility to add any color for the vertical. If we add color code to keys then wouldn't be confusing for authors later on ?

File: blocks/measuring-impact-cards/measuring-impact-cards.js (hunk 2 above)
Comment (singhpritam18) [reply blocks/measuring-impact-cards/measuring-impact-cards.js:6]: Done

File: blocks/measuring-impact-cards/measuring-impact-cards.js (hunk 1 above)
Comment (singhpritam18) [reply blocks/measuring-impact-cards/measuring-impact-cards.js:25]: Incorporated as we discussed over call to have block name as prefix for Shortend Tag key and 'theme' as prefix for Verticals

File: blocks/measuring-impact-cards/measuring-impact-cards.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
+.measuring-impact-cards{
+    display: flex;
+    flex-flow: row wrap;
+    padding: 0;
+    gap: 20px;
+}
+
+.impact-card-wrapper{
```
Comment (thomsebastin) [inline blocks/measuring-impact-cards/measuring-impact-cards.css:8]: Can you please namespace all the selectors within block class name?

**Review by thomsebastin — CHANGES_REQUESTED** [review]

File: blocks/measuring-impact-cards/measuring-impact-cards.css (hunk 3 above)
Comment (singhpritam18) [reply blocks/measuring-impact-cards/measuring-impact-cards.css]: Done

**Review by thomsebastin — APPROVED** [review]

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
@singhpritam18 Please use variables for colors, sizes, spacing

**Comment by singhpritam18** [comment]
> @singhpritam18 Please use variables for colors, sizes, spacing

@mkbansal1 Done

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**